### ISSUE TITLE
Bash Script Needed

### ISSUE THREAD

**Issue by rocketbear27** [rocketbear27/cleanscript#1]
In order to have a repository that anyone on any machine can run. I need Bash scripts and other command line scripts.

**Comment by naclcaleb** [comment]
Could you give me  some specifications for these scripts?

It seems like you have some sort of PowerShell script, but you're waiting on bash for Linux and Mac?

**Comment by naclcaleb** [comment]
Also, just note that the bash script extension is `.sh`, not `.bash`

**Comment by rocketbear27** [comment]
Oops, my mistake on the wrong bash extension

**Comment by naclcaleb** [comment]
That's fine. I've made a fork, but I'm not sure what exactly you want this script to do yet. Could you explain?

**Comment by rocketbear27** [comment]
Specifications:
1. The script needs to clean **only** heavy and useless folders in `folder.sh` (usually, shortcut files)
2. The script needs to clean **only** heavy and useless files in `file.sh` (usually, shortcut files)
3. The last script needs to clean **all** heavy and useless files in the whole repo (usually shortcut files)

**Comment by naclcaleb** [comment]
Here are some things I still need to know:
1. Where should the script be located and what should I name it?
2. In reference to #1, do you mean the file `folder.sh`, when run, should only remove heavy and useless folders? Or that somehow the script should remove folders from the `folders.sh` file?
3. Same with #2...
4. What path am I removing files from? The user's Desktop folder?

**Comment by naclcaleb** [comment]
And...I was not meaning to reference issues with my "#1", I just meant number 1 in your list.

**Comment by rocketbear27** [comment]
1. the scripts should be in the `/shell` folder (you can rename it to bash)
2. Yes, the `folder.sh` will remove folders from the repo directory (heavy and useless)
3. The script, `file.sh` will remove files from the repo directory (heavy and useless)
4. a repo

**Comment by naclcaleb** [comment]
Just any repo? Or this specific one?
On second though, maybe I should have the user specify the path using command line arguments?

And lastly, what qualifies as "heavy and useless"?

**Comment by rocketbear27** [comment]
Just any repo or project directory. Yes, user specification of the path is a good idea. For example, files that are space-consuming but have no real use in a project, those files could take up and entire directory, so it would be better to just delete the whole directory. 

Heavy and Useless: 
1. Shortcuts
2. Thumbnails
3. Duplicate of file

**Comment by naclcaleb** [comment]
Alright, I'll work on it as soon as I can.

**Comment by naclcaleb** [comment]
I think I'll just write it in Python; it's much easier and more flexible.

**Comment by naclcaleb** [comment]
Question: what is the difference between a shortcut and a thumbnail?

For linux, a shortcut can be identified by a `.desktop` extension. How should I identify a thumbnail?

**Comment by rocketbear27** [comment]
Some images with this file extentions: JPEG, BMP, GIF, PNG, TIFF, AVI, PDF, PPTX, DOCX, HTML

**Comment by naclcaleb** [comment]
Ok, but some of those files are important to people. How are you planning to figure out which ones to remove?

**Comment by rocketbear27** [comment]
I think that we should leave thumbnails aside

**Comment by naclcaleb** [comment]
Great! I made a pull request, but I'm going to make a new one with some updates. Take a look at the `bash/README.md` file to see what I did.

**Comment by rocketbear27** [comment]
Nice!

**Comment by naclcaleb** [comment]
Is it safe to close this issue then?

**Comment by rocketbear27** [comment]
Yes, it is

**Comment by naclcaleb** [comment]
Ok. I can't close it, only you can.

**Comment by rocketbear27** [comment]
Whoops my mistake
